### PR TITLE
docker dev env

### PR DESCRIPTION
**Pull request by smalho01**

### CODE REVIEW

**Comment by smalho01** [comment]
I can't get get dtr to authenticate with pas locally or as part of this docker set up. There's no issues with the logic servers or with the PAS AEGIS tests so this is not a bug but user error on my part, I must be messing up the registration step or something. If anyone reviewing this is able to test authentication between pas and dtr and get it to work, please respond to this comment

File: package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "private": true,
   "scripts": {
     "start": "npm-run-all --parallel startBackendDev buildFrontendDev",
+    "startProd": "npm-run-all --parallel startBackendProd buildFrontendProd",
```
Comment (plarocque4) [inline package.json:7]: We haven't had any luck getting this run in parallel thing to work on LogicaHealth.

File: Dockerfile [hunk 2]
```diff
@@ -1,9 +1,6 @@
...
 WORKDIR /home/node/app
-COPY package*.json ./
-USER node
-RUN npm install
 COPY --chown=node:node . .
+RUN npm install
 EXPOSE 3005
-CMD [ "/home/node/app/run.sh" ]
\ No newline at end of file
+CMD npm run startProd
```
Comment (plarocque4) [inline Dockerfile:6]: LogicaHealth is using startTemplate, that is needed to make sure to read some environment variables. We should meet to discuss what might work here on LogicaHealth.

**Review by plarocque4 — APPROVED** [review]
Looks good and works on LogicaHealth. I'm not sure if you wan to try running locally again now that some some changes were made.

**Pull request merged**